scripts/ct/run.sh

```diff
@@ -202,6 +202,9 @@ for dep in ${CT_DEPS}; do
         rocketmq)
             FILES+=( '.ci/docker-compose-file/docker-compose-rocketmq.yaml' )
             ;;
+        rocketmq_ssl)
```

Comment:
Nit: we could include this new docker compose file in the existing `rocketmq` entry.

Reply:
Fixed: [0cbbae9](https://github.com/emqx/emqx/pull/13274/commits/0cbbae9655fd28a1dd07bd553f274744e9b363de)